### DISCUSSION TITLE
Is this course focused on core Solidity development or full-stack blockchain development?

### DISCUSSION THREAD

**Discussion by ANKIT131103** [Cyfrin/foundry-full-course-cu#2435] [Polls]
Is this course focused on core Solidity development or full-stack blockchain development?

**Comment by khandelwalmoksh** [comment]
More of Smart Contract Development, Smart Contract Security Researcher/Auditor. But yes, key Solidity concepts are touched

**Reply by EngrPips** [reply]
Indeed
